gnucash/gtkbuilder/dialog-preferences.glade

```diff
@@ -1748,7 +1748,7 @@ many months before the current month</property>
                 </child>
                 <child>
                   <object class="GtkRadioButton" id="pref/general/retain-type-days">
-                    <property name="label" translatable="yes">For</property>
+                    <property name="label" translatable="yes">For Day(s)</property>
```

Comment:
There's a 'days' label on the spin button so this will be redundant. @fellen is there a way to have context translations in GtkBuilder files?

Reply:
>  @fellen is there a way to have context translations in GtkBuilder files?

https://wiki.gnucash.org/wiki/I18N#Strings_in_Glade_files